luminosityleds/server
# LL-250

# Description
Changed docker image used for github login to match the new file architecture.

Subsequent # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

If using codespaces, make sure the ports 3000, 4000, and 8080 are reserved. Use the forwarded urls instead of localhost.
1. Insert the correct ``.env`` file into ``/src/routes``
2. Run ``docker compose build``
3. Run ``docker compose up``
4. Go to ``http://localhost:3000/`` (or the forwarded url on a codespace)
5. Log in with Github
6. If using codespaces, replace ``http://localhost:4000`` with the respective forwarded address
7. Check for json output

# Checklist (when relevant):


    [TypeScript](https://google.github.io/styleguide/tsguide.html)
[Python](https://peps.python.org/pep-0008/)
[Schema](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/forefront-2010/ee652242(v=vs.100))
- [x] I have included a reviewer on the pull request [*Replace this text with the reviewer*]
